tools/genenotebook/genenotebook_build.xml

```diff
@@ -9,12 +9,25 @@
 @START_GNB@
 
 #for genome in $genomes:
-    genenotebook add genome @CONNECT_INFO@ --name '${genome.name}' '${genome.genome}';
+    genoboo add genome @CONNECT_INFO@ --name '${genome.name}' ${genome.public} '${genome.genome}';
```

Comment:
I keep reading bonobo :monkey:

Reply:
ahah, will think about it everytime I run it now :D
Thanks for the merge!